### PR TITLE
docs(agent): add new hub options RBK-201

### PR DESCRIPTION
**Pull request by 317brian**
https://s.armory.io/YEuORpPX

[RBK-201]
* Adds new connection options for hub
* Adds callout at the top of the table for saying don't use clouddriver in .6+
* Removes the badges for v .3 and .4

[RBK-201]: https://armory.atlassian.net/browse/RBK-201

### CODE REVIEW

**Review by aimeeu — APPROVED** [review]

File: content/en/docs/armory-agent/agent-options.md [hunk 1]
```diff
@@ -133,4 +133,6 @@ kubernetes:
 
 ## Configuration options
 
+> Note that all of the `clouddriver.` configs are not supported in v 0.60 and later. Use the equivalent `hub.` config.
```
Comment (german-muzquiz) [inline content/en/docs/armory-agent/agent-options.md]: Typo: version `0.6.0`

**Review by german-muzquiz — APPROVED** [review]
Looks good to me, but let's wait on merging until we release version `0.6.0`. 
There have been recent changes in the scope and roadmap of agent that will probably make it work with both hub and clouddriver at the same time, so the `clouddriver.*` settings could be ressurected from the dead.

**Comment by 317brian** [comment]
Converting to draft until we're ready for this.

**Comment by 317brian** [comment]
Since we aren't merging this until 0.6.0 is out, I will also make the following changes in this PR:

* Remove the note about the helm install not installing a stable image 
* Moving the helm install to the primary tab so that it's displayed first. (Currently the manual kubectl install is displayed first since it installs a stable release)

~Alternatively, we can remove the manual install completely if we want.~ 

FYI @danielGz

**Pull request merged**